metals/src/main/scala/scala/meta/internal/metals/BuildTargets.scala

```diff
@@ -31,7 +31,7 @@ final class BuildTargets() {
     workspace = newWorkspace
   }
   private var tables: Option[Tables] = None
-  private val sourceDirectoriesToBuildTarget =
+  private val sourceItemsToBuildTarget =
```

Comment:
I was thinking about renaming the `sourceItems` to just `sources` whenever we find it ;)